pkgs/development/python-modules/os-client-config/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  glanceclient,
+  oslotest,
+  openstacksdk,
+  stestr,
+  setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "os-client-config";
+  version = "2.1.0";
+  pyproject = true;
+
+
```

Comment:
```suggestion

```

Reply:
Thanks your big review !  
I am sorry you have to go through all of these files and found a lot because I forgot to push the latest commits I have done for this MR !